### PR TITLE
 extensions: add nbft extension (jsc#PED-3132) 

### PR DESCRIPTION
**Pull request by cfconrad**
This extension imports configuration from the NBFT,
according to the NVM Express Boot Specification 1.0
and convert it to the correspondig wicked configuration.

### CODE REVIEW

File: testing/Makefile.am [hunk 1]
```diff
@@ -47,6 +47,7 @@ EXTRA_DIST			= ibft xpath \
 
 TESTS				= socket-mock-test	\
 				  bitmask-test		\
-				  bitmap-test
+				  bitmap-test		\
+				  test-nbft
```
Comment (mtomaschewski) [inline testing/Makefile.am:51]: Why to follow conventions -- everything else is using `*-test` -- why nbft `test-nbft`?
Please also make sure that `make distcheck` works.

Comment (cfconrad) [reply testing/Makefile.am]: * renamed it to `nbft-test.sh`
 * fixed `make distcheck` thx @mwilck

Comment (mtomaschewski) [reply testing/Makefile.am]: OK, `./autogen.sh && make rpmpackage` works now:
```
make[2]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird betreten
make[3]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird betreten
PASS: socket-mock-test
PASS: bitmask-test
PASS: bitmap-test
PASS: nbft-test.sh
============================================================================
Testsuite summary for wicked 0.6.71
============================================================================
# TOTAL: 4
# PASS:  4
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[3]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird verlassen
```

**Review by mtomaschewski — APPROVED** [review]
OK, LGTM now, thanks!

**Pull request merged**